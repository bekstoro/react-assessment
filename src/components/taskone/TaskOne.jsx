import React from 'react';
import { Heading, Task } from '@components/styled-components/Task';
import UserList from '@components/taskone/UserList';

const TaskOne = () => (
  <Task>
    <Heading>Task One</Heading>
    <UserList />
  </Task>
);

export default TaskOne;
