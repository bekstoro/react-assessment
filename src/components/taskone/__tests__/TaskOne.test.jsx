import React from 'react';
import { render } from '@testing-library/react';

import TaskOne from '@components/taskone/TaskOne';

describe('TaskOne', () => {
  it('TaskOne renders correctly', () => {
    const { container } = render(<TaskOne />);
    expect(container).toMatchSnapshot();
  });
});
