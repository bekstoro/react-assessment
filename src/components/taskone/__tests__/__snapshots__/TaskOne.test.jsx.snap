// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TaskOne TaskOne renders correctly 1`] = `
<div>
  <div
    class="Task-sc-7z955n-0 dBJrLg"
  >
    <h1
      class="Task__Heading-sc-7z955n-1 hiEtuP"
    >
      Task One
    </h1>
    <div>
      <div>
        Filter:
        <input
          placeholder="Enter username"
          type="text"
          value=""
        />
      </div>
      <div
        class="UserList__Users-g96ac2-2 fpNOEP"
      />
    </div>
  </div>
</div>
`;
